packages/graph/hash_graph/lib/graph/src/knowledge/entity/query.rs

```diff
@@ -422,7 +386,7 @@ mod tests {
 
     #[test]
     fn deserialization() {
-        assert_eq!(deserialize(["version"]), EntityQueryPath::Version);
+        assert_eq!(deserialize(["version"]), EntityQueryPath::DecisionTimeFrom);
```

Comment:
Should `"version"` be `LowerTransactionTime`?

Reply:
This was addressed in the next PR 🙂 (The test failed)